src/Microsoft.DotNet.Interactive/KqlDiscoverabilityKernel.cs

```diff
@@ -25,6 +25,9 @@ public KqlDiscoverabilityKernel() : base(DefaultKernelName)
             "MsKqlKernel"
         };
         KernelInfo.LanguageName = "KQL";
+        KernelInfo.Description = $"""
+                            Query a Kusto database
+                            """;
```

Comment:
nit: Indents - not sure whether this will end up including extra whitespace in the literal (does it?)

Reply:
It does not.